docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -52,21 +52,33 @@ There should also be a call to `CustomFunctions.associate("id", functionName);`
 
 #### id
 
-The id is used as the invariant identifier for the custom function stored in the document. It should not change.
+The `id` is used as the invariant identifier for the custom function stored in the document. It should not change.
 
-* If id is not provided, the JavaScript/TypeScript function name is converted to uppercase and disallowed characters are removed.
-* The id must be unique for all custom functions.
+* If `id` is not provided, the JavaScript/TypeScript function name is converted to uppercase and disallowed characters are removed.
+* The `id` must be unique for all custom functions.
 * The allowed characters are limited to: A-Z, a-z, 0-9, underscores (\_), and period (.).
 
 #### name
 
-Provides the display name for the custom function.
+Provides the display `name` for the custom function.
 
 * If name is not provided, the id is also used as the name.
 * Allowed characters: Letters [Unicode Alphabetic character](https://www.unicode.org/reports/tr44/tr44-22.html#Alphabetic), numbers, period (.), and underscore (\_).
 * Must start with a letter.
 * Maximum length is 128 characters.
 
+### description
```

Comment:
This heading looks a bit odd interspersed between the other tags (as though it should also have a tag). I'd move this up to be the first discussion item before getting into tags. Two reasons.
1. It won't be in between the other tags.
2. JSDoc almost always starts with the description. And we've been doing the same with our samples in Script lab. I think we should consistently do this with all samples.